src/bridge-deployer/README.md

```diff
@@ -34,7 +35,9 @@ The general syntax for using the Bridge Deployer is:
 
 - `--identity <IDENTITY_PATH>`: Path to the identity file
 - `--private-key <PRIVATE_KEY>`: Private Key of the wallet to use for the transaction (can be provided as an environment variable `PRIVATE_KEY`)
-- `--evm-network <EVM_NETWORK>`: EVM network to deploy the contract to (e.g. "mainnet", "testnet", "localhost")
+- `--ic-network <IC_NETWORK>`: IC network to deploy the contract to (e.g. "mainnet", "testnet", "localhost")
```

Comment:
If I got it right
- ic-network is for the bridge canister deployment (i.e. not the contract)
- evm-principal is for the contract deployment on the evm side using the evm-canister (i.e. not the bridge)
- evm-rpc is for the contract deployment on the evm side using a generic eth node

Reply:
I agree with Francesco, this renaming is quite confiusing. `ic-network` is either IC mainnect or local with configured replica URL. `evm-network` is either mainnet (`https://mainnet.bitfinity.network`), testnet (`https://testnet.bitfinity.network`) or local (with a way to connect to local canister, probably with evm canister principal). I think the code uses the value in the second sense, but the name is from the first one.

Reply:
@veeso this doc is to be fixed. As I wrote before the terms "contract" and "bridge" are confusing. For example, here it should be `bridge` not `contract`:
```
`--ic-network <IC_NETWORK>`: IC network to deploy the contract to
```

Finally, if we use `evm-network` for evm, why for the bridge we use `ic-network` instead of `bridge-network` ?